### DISCUSSION TITLE
is it possible to change where HyperHDR reads the temperature from?

### DISCUSSION THREAD

**Discussion by slenderman-poon** [awawa-dev/HyperHDR#841] [Q&A] [answered]
![Screenshot 2024-05-02 134201](https://github.com/awawa-dev/HyperHDR/assets/36755376/c8a41fd5-9075-4a1a-bbde-1281c6b2ca73)

screen shot pretty much explains it... does anyone know if it's possible? or should I log an issue?

best regards

**Comment by awawa-dev** [comment, marked as answer]
Hi
You can change this line and recompile the app: https://github.com/awawa-dev/HyperHDR/blob/dd19ae6539643775c79ba9e9c9667408ff9bfdf4/sources/utils/SystemPerformanceCounters.cpp#L357C21-L357C58
I don't plan to make changes to support custom/non-standard sensors directly as a workaround for broken sensors. In your system /sys/class/thermal/thermal_zone0/temp is present anyway (otherwise HyperHDR would not show the temperature reading) so there is no reason to test other paths.

**Comment by slenderman-poon** [comment]
Hi awawa-dev,
huge thanks for the speedy response. I'll check it out.

thanks for making HyperHDR available, its awesome and works great :)

**Comment by slenderman-poon** [comment]
found an alternate solution incase anyone else has the same issue (maybe it's an intel NUC issue 🤷‍♂️)...
adding "thermal.off=1" to your grub kernel parameter will totally disable acpitz-acpi-0 which for me is what was symlinked to thermal_zone0
hope this helps someone else in a similar predicament.
 
![image](https://github.com/awawa-dev/HyperHDR/assets/36755376/455db23e-b198-484a-b08f-d3a74462bbe5)
